### PR TITLE
Replace JSHint with ESLint

### PR DESCRIPTION
**Pull request by aureliosaraiva**
For https://github.com/emberjs/guides/issues/1951 https://github.com/emberjs/guides/issues/2003

### CODE REVIEW

File: source/glossary/web-development.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ Both LESS and Sass are types of CSS preprocessor markup intended to give you muc
 [Sass/Less Comparison](https://gist.github.com/chriseppstein/674726)
 
 
-## Linter, linting, jslint, jshint
+## Linter, linting, jslint, jshint, eslint
```
Comment (locks) [inline source/glossary/web-development.md]: I was wondering, maybe we stick to "Linter, linting"  and list the actual linters in the text as examples of linters, and we can even link to the projects?

File: source/tutorial/service.md [hunk 2]
```diff
@@ -29,6 +29,13 @@ Add your new API key to the application by stopping the server and restarting it
 GOOGLE_MAPS_API_KEY=<your key here> ember s
 ```
 
+If you're running Windows
```
Comment (locks) [inline source/tutorial/service.md]: whoops?

File: source/testing/acceptance.md [hunk 3]
```diff
@@ -223,23 +223,22 @@ export default Ember.Test.registerAsyncHelper('addContact',
...
+`tests/helpers/start-app.js`. In `tests/.eslintrc.js` you need to add it in the
+`globals` section, otherwise you will get failing ESLint tests:
+
+```tests/.eslintrc.js{-4,+5,+6,+7,+8,+9,+10}
+module.exports = {
+  env: {
+    embertest: true
+  }
+  },
+  globals:{
```
Comment (locks) [inline source/testing/acceptance.md]: missing space

File: source/tutorial/service.md (hunk 2 above)
Comment (aureliosaraiva) [inline source/tutorial/service.md]: https://github.com/emberjs/guides/issues/2003

I'll put in another PR

File: source/glossary/web-development.md (hunk 1 above)
Comment (aureliosaraiva) [inline source/glossary/web-development.md]:
```
## Linter, linting
A validation tool which checks for common issues in your JavaScript. You'd usually use this in your build process to enforce quality in your codebase. A great example of something to check for: *making sure you've always got your semicolons*.

An example of some of the options you can configure:
[ESLint](http://eslint.org/docs/rules/)
[JSlint](http://jshint.com/docs/options/)
```

It's that what you thing? If this is so, I think it's great.

Comment (locks) [inline source/glossary/web-development.md]: Yup :)

**Comment by toddjordan** [comment]
Thanks so much @aureliosaraiva !!!!

**Pull request merged**